src/CalculatorUITests/StandardModeFunctionalTests.cs

```diff
@@ -47,15 +58,18 @@ public void TestInit()
             {
                 page.ClearAll();
             }
+            page.AppName.Click();
         }
 
         [TestCleanup]
         public void TestCleanup()
         {
+            page.StandardOperators.EqualButton.Click();
```

Comment:
>page.StandardOperators.EqualButton.Click(); [](start = 12, length = 43)

Is this necessary?  I think ClearAll handles most of this...

Reply:
It commits to the history panel so that the history delete button shows up. However, I think removing this and using the page source before clicking the button makes more sense.

Reply:
Change will appear in next commit